recipes/sqsgenerator/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "sqsgenerator" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dnoeger/sqsgenerator/archive/master.zip
```

Comment:
Don't use master. Use a stable release

Reply:
It doesn't have to be stable, just a release that isn't an rc or pre-release